app-crypt/nitrocli/nitrocli-0.1.3.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CRATES="
+cc-1.0.25
+hid-0.4.1
+hidapi-sys-0.1.4
+libc-0.2.45
+nitrocli-0.1.3
+pkg-config-0.3.14
+"
+
+inherit cargo
+
+DESCRIPTION="A command line tool for interacting with the Nitrokey Storage."
+HOMEPAGE="https://github.com/d-e-s-o/nitrocli"
+SRC_URI="$(cargo_crate_uris ${CRATES})"
+RESTRICT="mirror"
```

Comment:
Please follow the default skel structure, its getting hard to read this ebuild when all these variables are together like this.